### ISSUE TITLE
Limit the numbers of running test for the whole testkube server and place them into queue?

### ISSUE THREAD

**Issue by naveenreddymanukonda** [kubeshop/testkube#4225]
**Is your feature request related to a problem? Please describe.**
I had a use case where the user clicks on the run button and in the backend, I am triggering this test, and the number of users expected to click on the run button at any given point of time is 10-15 

**Describe the solution you'd like**
Need to have the option to select the number of tests that can be executed at once and the remaining should be in the queue

Currently, when we try to execute, these tests are executed in parallel and show the status as skipped or the job's getting terminated.

can, You please work this on high priority

**Comment by rangoo94** [comment]
I think it's a duplicate of https://github.com/kubeshop/testkube/issues/3584

**Comment by TheBrunoLopes** [comment]
Hey @naveenreddymanukonda 
we will work on this issue as soon as possible.
Closing it to not have duplicated issues.
Please track it on  https://github.com/kubeshop/testkube/issues/3584
